internal/provider/clickhouse_cluster_resource.go

```diff
@@ -565,6 +586,11 @@ func (r *ClickhouseClusterResource) Update(ctx context.Context, req resource.Upd
 	}
 	dcOperation, err := r.svc.Update(ctx, rq)
 	if err != nil {
+		if e, ok := status.FromError(err); ok && e.Code() == codes.FailedPrecondition && e.Message() == "no changes detected" {
+			resp.Diagnostics.AddWarning("no changes detected", err.Error())
```

Comment:
Why do you hide these situations?
I expect that if we tried to make update and failed with the error "no changed detected" it means that we had drift between local terraform model and model from the server and we failed to build the difference between them, which looks like internal error, not like a warning.

Would you mind to describe a case when you think it's ok to supress the error?

Reply:
It was for debug, I can remove it